src/components/home/homeBottomSection/homeBottomSection.tsx

```diff
@@ -0,0 +1,104 @@
+import { Button, Carousel, Col, Row } from "react-bootstrap";
+import PerfectScrollbar from "react-perfect-scrollbar";
+import "react-perfect-scrollbar/dist/css/styles.css";
+import projects from "./communities&standards.json";
+
+const HomeBottomSection = () => {
+  return (
+    <Row className="mx-2 mb-3">
+      <Col>
+        <h4 className="fw-bold fs-3 p-3 pb-1">Our Standards</h4>
+        <hr className="mx-3 border-primary mb-5" />
+        <Carousel indicators={false} variant="dark" interval={null}>
+          {projects
+            .map((value) => ({ value, sort: Math.random() }))
+            .sort((a, b) => a.sort - b.sort)
+            .map(({ value }) => value)
+            .map((x, idx) => (
```

Comment:
The line between title and subtitle should be light green

Reply:
Done